init.js

```diff
@@ -36,8 +36,8 @@ function init() {
       state.phase = "failed"
     } else {
       // Proceed to initialize the application
-      if (network > 2) { network = 3; }
-      chain.env = [, "live", "morden", "develop"][network]
+      if (network > 3) { network = 4; }
```

Comment:
That's for testing with TestRPC, which gives a really large number for its network id. So we just clamp that as index 4, then we mark it as `develop` in the next line:

`chain.env = [, "live", "morden", "ropsten", "develop"][network]`